Update docs for Release01

afoglia opened the pull request:
These are the updated docs.  I've dug in more, and the extra factor of the number of dimensions M does actually change the BIC computation.  It multiplies the number of clusters K(\phi).  This actually differs from the log-likelihood formula given in Pelleg and Moore at the bottom of page 4.  I'll think about it overnight, but I don't see where my math is wrong.

bobhancock committed:
Edge case when variance is zero, log likelihood returns -Inf

bobhancock committed:
Added loglikelihMoore; tests broken

bobhancock committed:
Corrected DataCentroids, test Kmeansp in progress.

bobhancock committed:
Removed extraneous tests for len([]) == 0

bobhancock committed:
Debug version.  Bob's Your Uncle smoothing.

bobhancock committed:
Instituted normalized smoothing for variance.

bobhancock committed:
Cleaned up comments.

bobhancock committed:
Consolidated manip.go and km.go.

danielhfrank committed:
uncomment and fix broken test by removing err return from DataCentroids

danielhfrank committed:
fix 2pi error, and add test to guarantee fix

bobhancock committed:
BIC corrected to multiply log(R).

bobhancock committed:
Corrected CalcBIC test.

bobhancock committed:
Merge pull request #16 from danielhfrank/cc_tests

Reviewed

bobhancock committed:
Additive constant in Loglikelihood is Nextafter(0, 1)

bobhancock committed:
Merge branch 'master' of https://github.com/bobhancock/goxmeans

bobhancock committed:
Merged with models.

bobhancock committed:
Added bisectcc CentroidChooser to Models()

bobhancock committed:
Merge branch 'models'

bobhancock committed:
kmeansp: removed passing of CentPointDist.

bobhancock committed:
Models: bisection concurrent

bobhancock committed:
Comment and README updates.

bobhancock committed:
Merge branch 'models'

bobhancock committed:
fixed and renamed boundaries()

bobhancock committed:
kmeansp: Replaced FiltColMap with slice.

bobhancock committed:
Maide model fields exportable.

bobhancock committed:
Merge branch 'models'

bobhancock committed:
Exported Numpoints() and Numcentroids()

bobhancock committed:
EuclidDist.CalcDist mod matrix to avoid allocation

bobhancock committed:
Merge branch 'models'

bobhancock committed:
Multi-level bisection

bobhancock committed:
Xmeans: only k

bobhancock committed:
Added functest and gendata

bobhancock committed:
cleanup bisect

bobhancock committed:
Merge branch 'bisect'

bobhancock committed:
Inline bisect

bobhancock committed:
remove inline bisect

bobhancock committed:
functest output

bobhancock committed:
rm functest and gendata

bobhancock committed:
Merge branch 'optimize'

bobhancock committed:
kmeansp after bisection

bobhancock committed:
Xmeans accepts centroids arg

bobhancock committed:
TestXmeans fixed

bobhancock committed:
Merge branch 'optimize'

bobhancock committed:
Pass clusters, return model

bobhancock committed:
pass cluster, return model

bobhancock committed:
Merge branch 'optimize'

bobhancock committed:
added examples

bobhancock committed:
float64 dataset

bobhancock committed:
log append

bobhancock committed:
int and float datasets

bobhancock committed:
remove debug prints

bobhancock committed:
Merge branch 'optimize'

danielhfrank committed:
rm unused method and test

danielhfrank committed:
add more comments to centroid choosers

bobhancock committed:
Removed err return from kmeans

bobhancock committed:
README Overview

bobhancock committed:
Merge branch 'optimize'

bobhancock committed:
Error checking for k <= kmax

bobhancock committed:
Error checking for k <= kmax

bobhancock committed:
test for k <= kmax

bobhancock committed:
Merge branch 'optimize'

bobhancock committed:
Merge pull request #21 from danielhfrank/df_cleanup

Cleanup methods from Dan

bobhancock committed:
Merge branch 'master' of https://github.com/bobhancock/goxmeans

bobhancock committed:
Added license

bobhancock committed:
README installation instructions

bobhancock committed:
README Ph.D

afoglia committed:
Add new version of notes

Also included latex and bibtex input files.

afoglia committed:
Update pdf

bobhancock commented:
Using the PDF as the reference document.

Page 1, "When the computed under the maximum likelihood" should be "When computed under the maximum likelihood"

Page 3 "Each individual cluster is determined a spherical gaussian" should this be "Each individual cluster is determined [as, by] a spherical gaussian"

Page 3 "Substitution (12) into (11) yields"  should this be "Substitution of (12) into (11) yields"

Nice explanation of the two hypotheses.

afoglia committed:
Fix typos

Most found by Bob in pull request.
A few spelling corrections.
A couple found by lacheck.

afoglia commented:
Fixed, and found a ran lacheck and aspell as well.

On Tue, Oct 2, 2012 at 2:04 PM, Bob Hancock notifications@github.comwrote:

> Using the PDF as the reference document.
> 
> Page 1, "When the computed under the maximum likelihood" should be "When
> computed under the maximum likelihood"
> 
> Page 3 "Each individual cluster is determined a spherical gaussian" should
> this be "Each individual cluster is determined [as, by] a spherical
> gaussian"
> 
> Page 3 "Substitution (12) into (11) yields" should this be "Substitution
> of (12) into (11) yields"
> 
> Nice explanation of the two hypotheses.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/bobhancock/goxmeans/pull/23#issuecomment-9080690.

## 

--Anthony
